RecoMuon/L2MuonSeedGenerator/src/L2MuonSeedGeneratorFromL1T.cc

```diff
@@ -124,246 +157,804 @@ void L2MuonSeedGeneratorFromL1T::produce(edm::Event& iEvent, const edm::EventSet
 
   auto output = std::make_unique<L2MuonTrajectorySeedCollection>();
   
+  if(matchType>4 || sortType>3) {
+    throw cms::Exception("Configuration") << "Wrong MatchType or SortType" << endl;
+  }
+
   edm::Handle<MuonBxCollection> muColl;
   iEvent.getByToken(muCollToken_, muColl);
   LogTrace(metname) << "Number of muons " << muColl->size() << endl;
 
-  edm::Handle<edm::View<TrajectorySeed> > offlineSeedHandle;
-  vector<int> offlineSeedMap;
-  if(useOfflineSeed) {
-    iEvent.getByToken(offlineSeedToken_, offlineSeedHandle);
-    LogTrace(metname) << "Number of offline seeds " << offlineSeedHandle->size() << endl;
-    offlineSeedMap = vector<int>(offlineSeedHandle->size(), 0);
+  /*
```

Comment:
Please remove the commented out code that you included for your debug (or use some LOgDebug if you want to maintain here for possible future needs)